src/opnsense/mvc/app/models/OPNsense/Base/FieldTypes/BaseField.php

```diff
@@ -363,6 +363,33 @@ public function __toString()
         return (string)$this->internalValue;
     }
 
+    /**
+     * return field current value
+     * @return null|string field current value
+     */
+    public function getCurrentValue(): ?string
+    {
+        return trim((string)$this->internalValue);
```

Comment:
I'm pretty sure trim() is not a good idea even if the impact is subtle. We had trim() in password processing before which isn't ideal for example.

Reply:
no worries, let's remove it